src/main/java/org/springframework/integration/kafka/dsl/KafkaProducerMessageHandlerSpec.java

```diff
@@ -258,6 +260,24 @@ public S sendTimeout(long sendTimeout) {
 	}
 
 	/**
+	 * Add a default header mapper to map spring messaging headers to Kafka headers.
+	 * @return the spec.
+	 */
+	public S headerMapper() {
+		return headerMapper(new DefaultKafkaHeaderMapper());
+	}
+
+	/**
+	 * Specify a header mapper to map spring messaging headers to Kafka headers.
+	 * @param mapper the mapper.
+	 * @return the spec.
```

Comment:
`@since 2.3.1` + `@see KafkaProducerMessageHandler#setHeaderMapper` ?

Reply:
This is 3.0 only - no DSL in 2.3.

Reply:
Doh. Yes. Thanks for the pointer!